### PR TITLE
Cluster policy controller again

### PR DESCRIPTION
**Pull request by sallyom**
opening here because i lost the ownership of commits in this:  https://github.com/openshift/cluster-kube-controller-manager-operator/pull/287  
@deads2k 
- adds cluster-policy-controller container to static pod.
related PRs
* rolebinding/role: https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/120
* lock  NamespaceSCCAllocationController: https://github.com/openshift/openshift-controller-manager/pull/28
* lock ClusterPolicyController: https://github.com/openshift/cluster-policy-controller/pull/3
* CI https://github.com/openshift/release/pull/5075

### CODE REVIEW

File: bindata/v4.1.0/kube-controller-manager/leader-election-cluster-policy-controller-rolebinding.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  namespace: openshift-kube-controller-manager
+  name: system:openshift:leader-lock-cluster-policy-controller
+roleRef:
+  kind: Role
+  name: system:openshift:leader-locking-cluster-policy-controller
```
Comment (deads2k) [inline bindata/v4.1.0/kube-controller-manager/leader-election-cluster-policy-controller-rolebinding.yaml]: you didn't create the role in this PR.  Please add it.

File: bindata/v4.1.0/kube-controller-manager/leader-election-cluster-policy-controller-rolebinding.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  namespace: openshift-kube-controller-manager
+  name: system:openshift:leader-lock-cluster-policy-controller
```
Comment (deads2k) [inline bindata/v4.1.0/kube-controller-manager/leader-election-cluster-policy-controller-rolebinding.yaml]: doesn't this name conflict with the openshift one?

Comment (deads2k) [reply bindata/v4.1.0/kube-controller-manager/leader-election-cluster-policy-controller-rolebinding.yaml]: > doesn't this name conflict with the openshift one?

ha, it doesn't.  Still make a different role and binding.

**Comment by deads2k** [comment]
this is important enough to bring in some debt for.  Please create the followup PR to cleanup the role and binding.  Also, once this merges, open a PR to remove the controllers from the openshift-controller-manager.

/lgtm

File: bindata/v4.1.0/kube-controller-manager/leader-election-cluster-policy-controller-rolebinding.yaml (hunk 2 above)
Comment (sallyom) [reply bindata/v4.1.0/kube-controller-manager/leader-election-cluster-policy-controller-rolebinding.yaml]: yup, will clean that up

**Pull request merged**